provider_shopper/lib/models/cart.dart

```diff
@@ -40,4 +40,11 @@ class CartModel extends ChangeNotifier {
     // depend on it.
     notifyListeners();
   }
+
+  void remove(Item item) {
+    _itemIds.remove(item.id);
+    // This line tells [Model] that it should rebuild the widgets that
```

Comment:
No need to repeat the comment verbatim. The sample code should read like a tutorial of sorts.

Something like: 

```dart
// Don't forget to tell dependant widgets to rebuild _every time_ you change the model.
```

Reply:
ok I will add this one instead .